Compilation issue: assembly-plugin group id '113584762' is too big

While compiling a master branch I got following error:

```
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 01:51 min
[INFO] Finished at: 2018-01-04T21:00:31+01:00
[INFO] Final Memory: 174M/1993M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-assembly-plugin:3.0.0:single (make-assembly) on project ksql-package: Execution make-assembly of goal org.apache.maven.plugins:maven-assembly-plugin:3.0.0:single failed: group id '113584762' is too big ( > 2097151 ). Use STAR or POSIX extensions to overcome this limit -> [Help 1]

```

```
mvn --version                        
Apache Maven 3.3.9
Maven home: /usr/share/maven
Java version: 1.8.0_111, vendor: Oracle Corporation
Java home: /usr/lib/jvm/java-8-oracle/jre
Default locale: en_US, platform encoding: UTF-8
OS name: "linux", version: "4.6.3-040603-generic", arch: "amd64", family: "unix"
```

--- a/pom.xml
+++ b/pom.xml
@@ -271,6 +271,7 @@
                 <plugin>
                     <artifactId>maven-assembly-plugin</artifactId>
                     <configuration>
+                        <tarLongFileMode>posix</tarLongFileMode>
                         <descriptors>
                             <descriptor>src/assembly/package.xml</descriptor>
                         </descriptors>

